test/invoice_test.rb

```diff
@@ -6,8 +6,8 @@
 class InvoiceTest < MiniTest::Test
   attr_reader :parent, :invoice
   def setup
-    sales_engine = SalesEngine.new
-    @parent = InvoiceRepository.new('test/support/invoices.csv', sales_engine)
+    sales_engine = SalesEngine.new('test/support')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.